### PR TITLE
[1.29] repo: Dev v1.29.3

### PR DESCRIPTION
**Pull request by phlax**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -80,27 +80,8 @@ checks:
     - publish
     - verify
     required: true
-  windows:
```
Comment (adisuissa) [inline .github/config.yml:83]: Should disabling windows-CI be part of the backport?

**Comment by phlax** [comment]
@adisuissa seems like this branch is a bit broken - but i think we should push this through anyway - we need to reopen the branch to do anything else

File: .github/config.yml (hunk 1 above)
Comment (phlax) [reply .github/config.yml:83]: hmm - not sure we are offering that support

in the case of 1.29 i dont think we should either as its not been published anywhere

the problem with retaining support on the other branches is that ci runs in `main` by default so we would need to bring that back there (or hack some different ci path)

**Review by adisuissa — APPROVED** [review]
LGTM, thanks!

**Pull request merged**